src/actions/remove.js

```diff
@@ -55,13 +62,13 @@ async function removeFile({ params }) {
     .then(isUnlisted)
     .then(hasAccess(username))
     .then(assertUpdatable({}, true));
-  const provider = this.provider('remove', params, undefined, data);
-  const shouldDeleteCloneFile = !hasClone(data) || provider.canCopy();
 
-  if (!softDelete && !isClone(data) && shouldDeleteCloneFile) {
+  if (!softDelete && !isClone(data)) {
     // we do not track this
-    cleanupFileProvider(data.files, provider, log)
-      .catch((e) => log.fatal({ err: e }, 'failed to cleanup file provider for %s', filename));
+    // @todo why?
```

Comment:
why no await?

Reply:
files can take a long time to delete, but it doesn't really matter whether they have actually been deleted or not at the end of the day. so we schedule a delete, but if it doesn't go through it's ok